### DISCUSSION TITLE
Display larger image in m2o-Interface

### DISCUSSION THREAD

**Discussion by kamil-malinski** [directus/directus#14714] [Archive]
Hi!

I'm using the m2o-interface to link images to articles:

![CleanShot 2022-07-28 at 12 52 33@2x](https://user-images.githubusercontent.com/68733431/181488920-ba631607-ae58-4485-89eb-1cbb34161518.png)

What I would like to do is to display a much larger version of the image. I'm a little lost how to achieve that. Which display is being used for this thumbnail? Is it possible to create a new display that shows a larger version of the image?

Thanks!

Kamil

**Comment by rijkvanzanten** [comment]
Instead of the m2o interface, try using the Image interface. That has a nice big preview 🙂

**Reply by kamil-malinski** [reply]
thanks for your reply, but my "image" is a collection with several fields among the file itself, so I can not use the Image interface

is there any other way to achieve what I want to do?

**Reply by rijkvanzanten** [reply]
No, there isn't currently a system interface that renders it as a grid of cards or anything like that

**Reply by kamil-malinski** [reply]
would it be possible to use a custom interface there?

**Reply by rijkvanzanten** [reply]
Absolutely! In that case, you're in control of how it's presented to the client, so the sky's the limit 👍🏻

**Reply by kamil-malinski** [reply]
my problem is: I don't see a way to change the interface for rows of the m2o field:

<img width="914" alt="CleanShot 2022-07-28 at 23 30 56@2x" src="https://user-images.githubusercontent.com/68733431/181641161-dad2d815-c47e-40ac-9def-2c8e6d52eba5.png">

so how can I use a custom interface inside the m2o-interface? or do I have to create my own m2o-interface?

**Reply by rijkvanzanten** [reply]
You would have to build your own custom m2o interface, yeah
